app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (VaccineError,
+                        NotWearingMaskError)
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+    for visitor in friends:
+        try:
+            cafe.visit_cafe(visitor)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            masks_to_buy += 1
+    return f"Friends should buy {masks_to_buy} masks" if masks_to_buy \
+        else f"Friends can go to {cafe.name}"
```

Comment:
Breaking a line with ```\``` is making code less readable.
You can add parenthases ```()``` around your ```return``` so you can brake a line without ```\```, but it's your decision if you think it's not making your code less readable, don't change it.